packages/stargate/src/Stargate.ts

```diff
@@ -69,11 +70,25 @@ export const getSupportedTokenAddresses = async (
 ): Promise<Address[]> => {
   // Given a specific chain we would expect this function to return a list of supported token addresses
   if (_chainId === POLYGON_CHAIN_ID)
-    return ['0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174']
+    return [
+      '0x2791Bca1f2de4661ED88A30C99A7a9449Aa84174',
+      '0x7ceb23fd6bc0add59e62ac25578270cff1b9f619',
+    ] // [UDSC, WETH]
   if (_chainId === ARBITRUM_CHAIN_ID)
-    return ['0xaf88d065e77c8cC2239327C5EDb3A432268e5831']
+    return [
+      '0xaf88d065e77c8cC2239327C5EDb3A432268e5831',
```

Comment:
https://docs.arbitrum.io/bridge-tokens/concepts/usdc-concept

Do we want USDC or USDC.e?

Reply:
I think Stargate only accepts USDC (at least I don't see an entry for USDC.e)